nixos/release.nix

```diff
@@ -280,6 +280,7 @@ in rec {
   tests.openssh = callTest tests/openssh.nix {};
   tests.panamax = hydraJob (import tests/panamax.nix { system = "x86_64-linux"; });
   tests.peerflix = callTest tests/peerflix.nix {};
+  tests.postgresql.init = callTest tests/postgresql-init.nix {};
```

Comment:
Could you rename it to just `postgresql` test, we want everything related to postgresql to be in one test.